google-cloud-clients/pom.xml

```diff
@@ -420,6 +420,12 @@
         <artifactId>checker-compat-qual</artifactId>
         <version>2.5.5</version>
       </dependency>
+      <dependency>
```

Comment:
how come google-cloud-clients has a dependency on this?

Reply:
This is listed in the `dependencyManagement` section so that it can be inherited by the firestore module.